source/MissionAction.cpp

```diff
@@ -230,6 +230,21 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 			if(child.Size() >= 3)
 				paymentMultiplier += child.Value(2);
 		}
+		else if(key == "fine")
```

Comment:
Since a value is necessary, why not check hasValue? The only other keywords that don't check hasValue have some sort of default behavior when no value is given.
```suggestion
		else if(key == "fine" && hasValue)
```

Reply:
Good suggestion, changed as suggested.